src/benchmark_runner.cc

```diff
@@ -232,7 +234,12 @@ class BenchmarkRunner {
     // Base decisions off of real time if requested by this benchmark.
     i.seconds = i.results.cpu_time_used;
     if (b.use_manual_time) {
-      i.seconds = i.results.manual_time_used;
+      if (b.manual_time->cost_in_seconds_) {
```

Comment:
Here and elsewhere:
This "if custom units exist - use it, else use some hardcoded default" is kinda ugly,
and is pretty much guaranteed to be got wrong somewhere.
Instead, it should always use custom time units, which is by default just the current seconds.

Now that you put it like that, i guess i'm no longer _fully_ convinced that it is separate from the more custom timers work..

Reply:
I could make it so that by-default we serialize "seconds", "s", pass_through_function into ManualTime with a `customized_units == false`. That way, we just always go through `manual_time->cost_in_seconds` rather than checking the difference. It will also make printing easier too, so maybe that's the direction I go in?

Reply:
What i meant is, there should not be `customized_units` field.
The units should always be customized, with the default matching the current output.

Reply:
I made some changes! Mostly, the defaults are the new `BenchmarkTime` struct defaults. There are less conditionals now too, making it easier to reason about the code and not make future mistakes.

Reply:
Thanks. Need to try to review this..